WORKSPACE

```diff
@@ -35,7 +35,7 @@ bind(
 )
 
 # When updating envoy sha manually please update the sha in istio.deps file also
```

Comment:
Lizan,
   Please update istio.deps also at the same time.

Reply:
Done